package.json

```diff
@@ -54,6 +54,7 @@
     "webpack-dev-server": "^2.11.1"
   },
   "dependencies": {
+    "babel-plugin-annotate-pure-calls": "^0.2.2",
```

Comment:
Oops I just noticed this is a dep while it should be a devDep.
cc @Andarist

Reply:
Thanks for fixing.